frontend/src/pages/PendingAssessments/RequestItem/RequestTechnologies/TechnologyItem/TechnologyItemContainer.tsx

```diff
@@ -0,0 +1,23 @@
+import { FC, useState } from 'react';
+
+import { ITechnologyItemContainerProps } from 'pages/PendingAssessments/types';
+
+import TechnologyItem from './TechnologyItem';
+
+const TechnologyItemContainer: FC<ITechnologyItemContainerProps> = ({ technology }) => {
+  const [isFullInfoShown, setFullInfoShown] = useState(false);
+
+  const showFullInfo = () => setFullInfoShown(true);
+  const hideFullInfo = () => setFullInfoShown(false);
+
+  return (
+    <TechnologyItem
+      technology={technology}
+      isInfoShown={isFullInfoShown}
+      showInfo={showFullInfo}
+      hideInfo={hideFullInfo}
+    />
```

Comment:
```suggestion
  const [isFullInfoShown, toggleFullInfoShown] = useToggle();

  return (
    <TechnologyItem
      technology={technology}
      isInfoShown={isFullInfoShown}
      toggleFullInfoShown={toggleFullInfoShown}
    />
```

Maybe will be better

Reply:
done